ui/raidboss/data/05-shb/raid/e3s.js

```diff
@@ -105,39 +105,42 @@
       id: 'E3S Rip Current',
       netRegex: NetRegexes.headMarker({ id: '0017' }),
       suppressSeconds: 10,
-      alarmText: function(data, matches) {
-        if (matches.target != data.me && data.role == 'tank') {
-          return {
-            en: 'Tank Swap!',
-            de: 'Tankwechsel!',
-            fr: 'Tank swap !',
-            ja: 'スイッチ',
-            cn: '换T！',
-            ko: '탱 교대!',
-          };
-        }
-      },
-      alertText: function(data, matches) {
-        if (data.me == matches.target) {
-          return {
-            en: 'Tank Buster on YOU',
-            de: 'Tank buster auf DIR',
-            fr: 'Tank buster sur VOUS',
-            ja: '自分にタンクバスター',
-            cn: '死刑点名',
-            ko: '탱버 대상자',
-          };
-        }
-        if (data.role == 'healer') {
-          return {
-            en: 'Tank Busters',
-            de: 'Tank buster',
-            fr: 'Tank buster',
-            ja: 'タンクバスター',
-            cn: '死刑',
-            ko: '탱버',
-          };
-        }
+      alarmText: function(data, matches, output) {
```

Comment:
tankBusterSwap ?

Reply:
Sorry, it's not a response because it's got a special "plural tankbusters" for the healers.